alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java

```diff
@@ -74,10 +78,16 @@ private AltsChannelBuilder(String target) {
             .keepAliveTime(20, TimeUnit.SECONDS)
             .keepAliveTimeout(10, TimeUnit.SECONDS)
             .keepAliveWithoutCalls(true);
+    addGrpclbFactory(delegate);
     handshakerOptionsBuilder.setRpcProtocolVersions(
         RpcProtocolVersionsUtil.getRpcProtocolVersions());
   }
 
+  @VisibleForTesting
+  static void addGrpclbFactory(ManagedChannelBuilder<?> mcb) {
+    mcb.loadBalancerFactory(GrpclbLoadBalancerFactory.getInstance());
```

Comment:
My understanding is that @jiangtaoli2016 wants to get the GRPCLB functionality, so this is necessary.
Enabling SRV querying is a prerequisite and he is trying to do it in this PR.

Reply:
I understand SRV is a prerequisite. But assuming SRV is queried, is it also necessary to explicitly specify gRPC-LB? I thought that wasn't the case. And if it is the case, then it means there needs to be some work items to fix it.

Reply:
Oh, I didn't know that `AutoConfiguredLoadBalancerFactory` is already the default. It will use GRPCLB if any balancer address is provided by the naming system and grpclb is in the classpath.
@ejona86, you are right. It's not necessary to specify GRPCLB.
@jiangtaoli2016, you still need to have the grpclb target in your runtime dependencies.

Reply:
Cool. Let me just add grpclb dependency then.